### PR TITLE
Ankaios CLI "ank get workloads" does not list workload state "Stopping"

### PR DESCRIPTION
**Pull request by maturar**
Issues: #134 

# Definition of Done

The PR shall be merged only if all items mentioned in [CONTRIBUTING.md](https://github.com/eclipse-ankaios/ankaios/blob/main/CONTRIBUTING.md#how-to-contribute) have been followed. In case an item is not applicable as described, please provide a short explanation in the description.

- [x] All steps in [CONTRIBUTING.md](https://github.com/eclipse-ankaios/ankaios/blob/main/CONTRIBUTING.md#how-to-contribute) have been handled

### CODE REVIEW

**Comment by maturar** [comment]
The ticket has been reimplemented as agreed. I am putting the pull request on-hold. I am going to continue with the PR (unit tests, requirements) as soon as dependent PRs (#156 and #149)  are finished.

File: tests/stests/workloads/delete_workload_podman_kube.robot [hunk 1]
```diff
@@ -39,7 +39,7 @@ Test Ankaios Podman delete kube workload
     # Actions
     When user triggers "ank delete workload hello-k8s"
     # Asserts
-    Then the workload "hello-k8s" shall not exist
```
Comment (krucod3) [inline tests/stests/workloads/delete_workload_podman_kube.robot:42]: We can check the execution state here too and expect removed.

Same goes for the tests below. Please check them again.

**Review by krucod3 — CHANGES_REQUESTED** [review]
Reviewed, findings documented.

File: tests/stests/workloads/delete_workload_podman.robot [hunk 2]
```diff
@@ -39,12 +39,11 @@ Test Ankaios Podman remove workloads
     # Actions
     When user triggers "ank delete workload nginx"
     # Asserts
-    Then the workload "nginx" shall not exist
```
[the remark this answers is not in the document]
Comment (maturar) [reply tests/stests/workloads/delete_workload_podman.robot:42]: Fixed

File: tests/stests/workloads/delete_workload_podman_kube.robot (hunk 1 above)
Comment (maturar) [reply tests/stests/workloads/delete_workload_podman_kube.robot:42]: Fixed

**Review by krucod3 — APPROVED** [review]
LGTM 👍

**Pull request merged**